### ISSUE TITLE
Chatbot Integration Tests in Github Actions Pipeline

### ISSUE THREAD

**Issue by ksk385** [department-of-veterans-affairs/va.gov-team#40987]
## Who is the devops resource on your team?

@ksk385 

## Description
We need to run integration tests against our MSFT PVA Chatbot which lives in the VAEC (Virginia). These tests need to run in our CI pipeline on Github Actions. The test suite runs a vets-api mock server and we would need the PVA bot to reach out to the vets-api mocks while they run.

## Background/context 
Which runners should I use on Github Actions?
Are the runners publicly accessible on the internet? Which ports are exposed to incoming traffic?

## Technical notes
Do I need to run ngrok within the Github Action to make the mock server publicly accessible to the PVA bot? This is how we run these tests on our local machines.

---
## Tasks
- [ ] _What things need to happen?_

## Acceptance Criteria 
- [ ] _What are the specific things that must be true in order for this to be accepted?_

---
### Reminders
- [ ] Please attach your team label and any other appropriate label(s) (operations, devops, and needs-grooming will automatically be applied as part of the template)
- [ ] Please connect to an epic (this will typically be done by the Platform Operations PM or TL)

**Comment by oseasmoran73** [comment]
Hi @ksk385 

Do you still have questions pertaining this integration in Github Action?

If so, if I understand correctly, you run a mock server for the test suite and this PVA bot needs to access it, correct? Does the bot need a url provided?

I can't think of any reason why ngrok (I am not familiar with the tool) can't accomplish similarly to localhost. If PVA bot needs to be within dsva network we may be able to add `self-hosted` runners to reach it

**Comment by oseasmoran73** [comment]
Linking [slack](https://dsva.slack.com/archives/CBU0KDSB1/p1657219369530579?thread_ts=1651694414.428419&cid=CBU0KDSB1) thread

**Comment by oseasmoran73** [comment]
@ksk385 Have you identified where Infrastructure can help to unblock you?

**Comment by oseasmoran73** [comment]
@ksk385 bump

**Comment by oseasmoran73** [comment]
Per [ticket](https://github.com/department-of-veterans-affairs/va-virtual-agent/issues/283#issuecomment-1204489673) associated

> We are going to spin up a new card to make the architectural changes that MSFT team has recommended to get this going in the VA environment. Using the existing infrastructure it is not possible to run our Botium test suite as it stands.
>
> New Card: https://github.com/department-of-veterans-affairs/va-virtual-agent/issues/584

Closing this ticket. Please feel free to reopen or submit a new ticket

CC @jhouse-solvd
